components/ChipTextField/examples/MDCChipTextField.h

```diff
@@ -45,4 +84,12 @@
  */
 - (void)appendChipWithText:(nonnull NSString *)text NS_SWIFT_NAME(appendChip(text:));
```

Comment:
Should this return the chipView that was appended?

Reply:
Addressed in 0003b1507a139f7d11899e583f895bac70dd94e6